motionplan/tpspace/ptgGroupFrame.go

```diff
@@ -171,20 +180,19 @@ func (pf *ptgGroupFrame) Interpolate(from, to []referenceframe.Input, by float64
 	if len(to) != len(pf.DoF()) {
 		return nil, referenceframe.NewIncorrectInputLengthError(len(to), len(pf.DoF()))
 	}
-	if len(from) != len(pf.DoF()) {
-		// We also want to always support 2 inputs
-		return nil, referenceframe.NewIncorrectInputLengthError(len(from), len(pf.DoF()))
-	}
-
-	if from[ptgIndex].Value != to[ptgIndex].Value {
-		return nil, NewNonMatchingInputError(from[ptgIndex].Value, to[ptgIndex].Value)
-	}
-	if from[trajectoryAlphaWithinPTG].Value != to[trajectoryAlphaWithinPTG].Value {
-		return nil, NewNonMatchingInputError(from[trajectoryAlphaWithinPTG].Value, to[trajectoryAlphaWithinPTG].Value)
+	for i, input := range from {
```

Comment:
I'm confused why this part is changing dont you need the inputs to change for elements 2 and 3?

Reply:
Since the start and end configurations are now encoded in a single set of inputs, we only need one set of inputs for interpolation. So we want to ensure they are identical.

Reply:
That's weird to me; that would mean that start[end] and end[start] are ignored regardless of what they are or if they are valid. Seems like it would just make it easier to trip people up and pass in incorrect values.

Reply:
It seems equally weird to me to have a function with two arguments where you have to pass in the same thing for both of them.  But can agree my suggestion isnt without fault and am fine to move past this if you want to keep as is.

Reply:
I agree it's weird and not ideal, but as written it's not possible to misuse and there's no undefined behavior.